crates/matrix-sdk-common/src/linked_chunk/builder.rs

```diff
@@ -37,7 +37,7 @@ struct TemporaryChunk<Item, Gap> {
 #[allow(missing_debug_implementations)]
 pub struct LinkedChunkBuilder<const CAP: usize, Item, Gap> {
     /// Work-in-progress chunks.
-    chunks: HashMap<ChunkIdentifier, TemporaryChunk<Item, Gap>>,
+    chunks: BTreeMap<ChunkIdentifier, TemporaryChunk<Item, Gap>>,
```

Comment:
Should be a squashed with the previous commit if you ask me.

Reply:
i'll squash everything yeah